### PR TITLE
Add message for the Debug Stick being unsupported

### PR DESCRIPTION
**Pull request by chandler05**
I think it would be a good idea to have a message explaining how the debug stick is not supported (and includes a link to the reddit post in which it was decided)

### CODE REVIEW

File: assets/messages.yml [hunk 1]
```diff
@@ -242,6 +242,18 @@ messages:
...
+      fillname: []
+  debug-stick:
+    - project: mc
+      name: Debug Stick
+      shortcut: stick
+      message: |-
+        *Thank you for your report!*
+        However, this issue won't be fixed.
+
+        Any issues regarding the Debug Stick are not accepted on the bug tracker according to a [reddit post|https://www.reddit.com/r/Minecraft/comments/7es23r/please_mojang_let_us_keep_the_debug_stick_it_has/dq7fjld/] from Mojang.
```
Comment (Marcono1234) [inline assets/messages.yml:255]:
```suggestion
        Any issues regarding the Debug Stick are not accepted on the bug tracker according to this [reddit post|https://www.reddit.com/r/Minecraft/comments/7es23r/please_mojang_let_us_keep_the_debug_stick_it_has/dq7fjld/] from Mojang.
```

**Comment by chandler05** [comment]
I edited the message a bit, it should be more clear now

File: assets/messages.yml [hunk 2]
```diff
@@ -242,6 +242,18 @@ messages:
...
+      fillname: []
+  debug-stick:
+    - project: mc
+      name: Debug Stick
+      shortcut: stick
+      message: |-
+        *Thank you for your report!*
+        However, this issue won't be fixed.
+
+        Any issues regarding the block states being created or changed using the Debug Stick are not accepted on the bug tracker according to this [reddit post|https://www.reddit.com/r/Minecraft/comments/7es23r/please_mojang_let_us_keep_the_debug_stick_it_has/dq7fjld/] from Mojang.
```
Comment (SPGoding) [inline assets/messages.yml:255]:
```suggestion
        Any issues regarding the block states being created or changed by using the Debug Stick are not accepted on the bug tracker according to this [reddit post|https://www.reddit.com/r/Minecraft/comments/7es23r/please_mojang_let_us_keep_the_debug_stick_it_has/dq7fjld/] from Mojang.
```

**Comment by NeunEinser** [comment]
To be extra sure, it might be worth asking Mojang if this is what they intend and in their best interest.
Personally, I think the message is okay as it is right now, though.

**Review by SPGoding — APPROVED** [review]

**Comment by Marcono1234** [comment]
> To be extra sure, it might be worth asking Mojang if this is what they intend and in their best interest.

Have the Mojang devs actually been asked regarding this message? I have not found something about it.
(If there has been discussion about it internally you can send me the link on the internal Mojira communication platform)

**Comment by Marcono1234** [comment]
The developers have been asked now (search for the link of this pull request; moderators only). The message appears to be fine.

**Comment by SPGoding** [comment]
Thank you!

**Pull request merged**